### DISCUSSION TITLE
copilot activation failed on vs code

### DISCUSSION THREAD

**Discussion by Amrat26728** [community/community#13555] [Copilot Conversations]
I tried to install github copilot on vs code and i authorized the github to vs code and when came  back to vs code it shown at the bottom menu it shows me that github copilot installation failed.
Now what can i do for installation?

**Comment by nokiae0106** [comment]
> I tried to install github copilot on vs code and i authorized the github to vs code and when came back to vs code it shown at the bottom menu it shows me that github copilot installation failed. Now what can i do for installation?

**Comment by Oskarin007009** [comment]
> I tried to install github copilot on vs code and i authorized the github to vs code and when came back to vs code it shown at the bottom menu it shows me that github copilot installation failed. Now what can i do for installation?

**Reply by Amrat26728** [reply]
please suggest me for installation successfully.
